### PR TITLE
Fix Typo in story.md

### PR DESCRIPTION
**Pull request by JcolemanNR**
My internal spell-checker noticed a typo when I was looking at git history

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/story.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ priority: ''
 ---
 
 ### Describe the story, please be clear on scope of the story.
-Please be conscice with the scope for the story.  
+Please be concice with the scope for the story.
```
Comment (nr-ahemsath) [inline .github/ISSUE_TEMPLATE/story.md]:
```suggestion
Please be concise with the scope for the story.  
```

Comment (nr-ahemsath) [reply .github/ISSUE_TEMPLATE/story.md]: https://www.merriam-webster.com/dictionary/concise

**Review by nr-ahemsath — APPROVED** [review]
👍

File: .github/ISSUE_TEMPLATE/story.md (hunk 1 above)
Comment (JcolemanNR) [reply .github/ISSUE_TEMPLATE/story.md]: Yeah, I noticed that after my initial commit. I see your brain has an auxiliary spell check module too! 😆

**Pull request merged**